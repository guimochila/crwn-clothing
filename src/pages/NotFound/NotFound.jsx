import React from 'react';
import { NotFoundContainer } from './NotFound.styles';

function NotFound() {
  return (
    <NotFoundContainer>
      <h1>Ooops! 404 </h1>
      <span>The page you are looking for does not exists.</span>
    </NotFoundContainer>
  );
}

export default NotFound;
