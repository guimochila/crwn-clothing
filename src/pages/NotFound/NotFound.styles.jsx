import styled from 'styled-components';

export const NotFoundContainer = styled.div`
  display: flex;
  flex-direction: column;
  justify-content: center;
  align-items: center;
  height: 60vh;

  h1 {
    letter-spacing: 2.5px;
  }

  span {
    font-size: 25px;
    letter-spacing: 2px;
  }
`;
