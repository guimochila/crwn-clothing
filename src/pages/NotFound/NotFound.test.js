import React from 'react';
import { shallow } from 'enzyme';
import NotFound from './NotFound';

it('it should render NotFound component', () => {
  expect(shallow(<NotFound />)).toMatchSnapshot();
});
