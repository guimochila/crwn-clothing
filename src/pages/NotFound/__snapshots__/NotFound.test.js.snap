// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`it should render NotFound component 1`] = `
<styled.div>
  <h1>
    Ooops! 404 
  </h1>
  <span>
    The page you are looking for does not exists.
  </span>
</styled.div>
`;
